### PR TITLE
feat(route): 黄健宏博客

### PR DESCRIPTION
**Pull request by stormbuf**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/huangz
```

## 新 RSS 检查列表 / New RSS Script Checklist

- [x] 新的路由 New Route
	- [x] 跟随 [[v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html)](https://docs.rsshub.app/joinus/script-standard.html) Follows [[v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
	- [x] 中文文档 CN
	- [ ] 英文文档 EN
- [ ] 全文获取 fulltext
	- [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
	- [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [[日期和时间](https://docs.rsshub.app/joinus/pub-date.html)](https://docs.rsshub.app/joinus/pub-date.html) [[date and time](https://docs.rsshub.app/en/joinus/pub-date.html)](https://docs.rsshub.app/en/joinus/pub-date.html)
	- [ ] 可以解析 Parsed
	- [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

### CODE REVIEW

File: docs/blog.md [hunk 1]
```diff
@@ -199,6 +199,11 @@ pageClass: routes
 
 <Route author="tpnonthealps" example="/fontstory" path="/fontstory" />
 
+## 黄健宏博客
+
+### 文章
+
+<Route author="stormbuf" example="/huangz" path="/huangz" radar="1"/>
```
Comment (TonyRL) [inline docs/blog.md:206]:
```suggestion
<Route author="stormbuf" example="/huangz" path="/huangz" radar="1"/>

```

File: lib/v2/huangz/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const got = require('got');
```
Comment (TonyRL) [inline lib/v2/huangz/index.js:1]:
```suggestion
const got = require('@/utils/got');
```

File: lib/v2/huangz/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'huangz.me': {
+        _name: 'huangz/blog',
```
Comment (TonyRL) [inline lib/v2/huangz/radar.js:3]: `_name` should match H2 heading of the docs
```suggestion
        _name: '黄健宏博客',
```

File: lib/v2/huangz/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        blog: [
+            {
+                title: 'huangz/blog',
```
Comment (TonyRL) [inline lib/v2/huangz/radar.js:6]:
```suggestion
                title: '文章',
```
`title` should match H3 heading of the docs

File: lib/v2/huangz/index.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        list.map((index, item) => {
+            const link = `${BLOG_URL}${$(item).find('a').attr('href')}`;
+
+            return ctx.cache.tryGet(
+                link,
+                () => ({
+                    title: $(item).find('a').text(),
+                    link,
+                }),
+                60 * 60 * 24
```
Comment (TonyRL) [inline lib/v2/huangz/index.js:24]: Please leave it empty to use the value of `cache.contentExpire` from config

**Comment by stormbuf** [comment]
done

**Pull request merged**